src/pages/LinkLandingPage/LinkLandingPage.tsx

```diff
@@ -108,14 +113,26 @@ const LinkLandingPage = ({ location, history }: RouteComponentProps) => {
             </Grid>
             <Grid item={true} xs={12}>
               <Fieldset legend={"To"}>
-                <TextField
-                  value={peer}
-                  label={"Peer"}
-                  helperText={"IPv4 Socket Address"}
-                  data-cy="peer-input"
-                  disabled={true}
-                  required={true}
-                />
+                <Grid item={true} xs={12} md={6}>
+                  <TextField
+                    value={peerId}
+                    label={"Peer"}
+                    helperText={"Peer Id"}
+                    data-cy="peer-input"
+                    disabled={true}
+                    required={true}
+                  />
+                </Grid>
+                <Grid item={true} xs={12} md={6}>
+                  <TextField
+                    value={addressHint}
+                    label={"Peer Address Hint"}
+                    helperText={"Address in multiaddress format"}
+                    data-cy="peer-input"
```

Comment:
:mega: Wrong `data-cy` label.
```suggestion
                    data-cy="address-hint-input"
```